rest_framework_tus/models.py

```diff
@@ -25,8 +25,8 @@ class AbstractUpload(models.Model):
 
     state = FSMField(default=states.INITIAL)
 
-    upload_offset = models.PositiveIntegerField(default=0)
-    upload_length = models.IntegerField(default=-1)
+    upload_offset = models.BigIntegerField(default=0)
```

Comment:
Hello, thank you for your patch. Could you add a "clean" method for this field to check for "only positive" values?